types/ckeditor__ckeditor5-core/src/plugin.d.ts

```diff
@@ -81,7 +81,7 @@ export default class Plugin implements Observable {
      *
      * **Note:** This method is optional. A plugin instance does not need to have it defined.
      */
-    destroy?(): void;
```

Comment:
this is defined on the base class. It does really make no sense (it's always there).
Is that base class kind of 'abstract' class, so people have to extend it and plug missing implementation (with 'destroy' being optional)

Reply:
I'm sorry but I do not understand your question. I hope this makes sense.

`Plugin` is an abstract class but `destroy` is defined in `Plugin`, so child classes are expected to call `super()` if they extend `destroy`.

https://github.com/ckeditor/ckeditor5/blob/72f503139a8fdf40b41cf26960a94b725f409338/packages/ckeditor5-core/src/plugin.js#L130

Reply:
sorry for not explaining this. the question was is 'Plugin' was the class without the 'destroy' body implementation, or body throwing exception (kind of abstract in JS). Now I can see that's not a case here. thx for answering!